datasets/wiki_auto/wiki_auto.py

```diff
@@ -0,0 +1,272 @@
+                    "simple": datasets.Features(
+                        {
+                            "simple_article_id": datasets.Value("int32"),
+                            "simple_article_title": datasets.Value("string"),
+                            "simple_article_url": datasets.Value("string"),
+                            "simple_article_content": datasets.Sequence(
+                                datasets.Features(
+                                    {
+                                        "simple_sentence_id": datasets.Value("string"),
+                                        "simple_sentence": datasets.Value("string"),
+                                    }
+                                )
+                            ),
+                        }
+                    ),
+                    "paragraph_alignment": datasets.Sequence(
+                        datasets.Features(
+                            {
+                                "normal_paragraph_id": datasets.Value("string"),
+                                "simple_paragraph_id": datasets.Value("string"),
+                            }
+                        )
+                    ),
+                    "sentence_alignment": datasets.Sequence(
+                        datasets.Features(
+                            {
+                                "normal_sentence_id": datasets.Value("string"),
+                                "simple_sentence_id": datasets.Value("string"),
+                            }
+                        )
+                    ),
+                }
+            )
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=features,
+            supervised_keys=None,
+            homepage="https://github.com/chaojiang06/wiki-auto",
+            license=_LICENSE,
+            citation=_CITATION,
+        )
+
+    def _split_generators(self, dl_manager):
+        my_urls = _URLs[self.config.name]
+        data_dir = dl_manager.download_and_extract(my_urls)
+        if self.config.name in ["manual", "auto"]:
+            return [
+                datasets.SplitGenerator(
+                    name=spl,
+                    gen_kwargs={
+                        "filepaths": data_dir,
+                        "split": spl,
+                    },
+                )
+                for spl in data_dir
+            ]
+        else:
+            return [
+                datasets.SplitGenerator(
+                    name="full",
+                    gen_kwargs={"filepaths": data_dir, "split": "full"},
+                )
+            ]
+
+    def _generate_examples(self, filepaths, split):
+        if self.config.name == "manual":
+            keys = [
+                "alignment_label",
+                "simple_sentence_id",
+                "normal_sentence_id",
+                "simple_sentence",
+                "normal_sentence",
+            ]
+            with open(filepaths[split]) as f:
```

Comment:
you need to specify the encoding here, for example `encoding="utf-8"`
same for the other `open` calls in `_generate_examples `